crates/bevy_ui/src/ui_node.rs

```diff
@@ -212,14 +214,19 @@ pub enum Direction {
     RightToLeft,
 }
 
-/// Whether to use Flexbox layout
+/// Whether to use a flexbox layout strategy
```

Comment:
```suggestion
/// Whether to use a Flexbox layout model.
```

Many sources, including the [specification], while inconsistently, wrote “Flexbox” capitalized, meaning it's probably a proper name (MDN writes it in lowercase but I would take the spec as authority). Also, both the specification and MDN refers to it as a “layout model”.

I also think the description could be changed for a more general one, but since we currently only support one layout model, there's no pressure to change it now.

[specification]: https://www.w3.org/TR/css-flexbox-1/#definite-sizes